src/views/teach/activities/home.jsx

```diff
@@ -16,17 +15,8 @@ const ActivitiesHome = () => (
             Each of these activities gives you a quick way to learn how
             to do new things with ScratchJr. They are listed here in
             order of simplest to hardest, but feel free to play around
-            in any order you&apos;d like! For more ScratchJr resources
-            developed by DevTech Research Group, click <a href="http://sites.tufts.edu/devtech/research/scratchjr/">here</a>.
+            in any order you&apos;d like!
```

Comment:
Add 
For more information and a brief introduction to ScratchJr see the Resources page
```suggestion
            in any order you&apos;d like! For more information and a brief introduction to ScratchJr see the <a href=" /teach/resources">Resources page</a>.
```
I added a static link for resources page - I don't remember if that's the right way to do it when using react-router - please fix if needed.